js/src/data/actions.js

```diff
@@ -261,14 +262,15 @@ export function* fetchShippingTimes() {
  * @throws Will throw an error if the request failed.
  */
 export function* upsertShippingTimes( shippingTime ) {
-	const { countryCodes, time } = shippingTime;
+	const { countryCodes, time, maxTime } = shippingTime;
```

Comment:
The new property is missing from the `AggregatedShippingTime` type.

https://github.com/woocommerce/google-listings-and-ads/blob/c1dffd305b34192e0d6daeb0e36571565c86827d/js/src/data/actions.js#L250-L256